src/main/webapp/index.html

```diff
@@ -203,77 +203,52 @@ <h5 class="display-3 text-center project-stat">
     <section class="section">
         <div class="container my-5">
             <div class="col">
-                <h4 class="font-weight-900 text-dark text-center">You could be a student, an expert or a volunteer. At SEF,
-                    we have something in store for everyone!</h4>
+                <h4 class="font-weight-900 text-center mb-md-5"> At SEF, we have something in store for everyone!</h4>
```

Comment:
The title should be bigger and do something better to make it stand out than the other titles like student, expert

Reply:
Done